api/v1alpha1/edgedevice_types.go

```diff
@@ -70,6 +70,19 @@ type Retention struct {
 type Storage struct {
 	S3 *S3Storage `json:"s3,omitempty"`
 }
+
+type OsInformation struct {
+
+	//Automatically upgrade the OS image
+	AutomaticallyUpgrade bool `json:"automaticallyUpgrade,omitempty"`
+
+	//CommitID carries information about commit of the OS Image
+	CommitID string `json:"commitID,omitempty"`
+
+	//HostedImagesURL carries the URL of the hosted images web server
+	HostedImagesURL string `json:"hostedImagesURL,omitempty"`
```

Comment:
Will you be changing that to reflect the fact that it points to os-tree commits rather than images?

Reply:
It points to the hosted Images service as it's mentioned in fleet management
In our case, RFE automation pipeline would be the hosted images service